Tools/OSD/osd_screen.cs

```diff
@@ -690,8 +713,17 @@ private void rbtSortAlphabetic_CheckedChanged(object sender, EventArgs e)
         private void pictureBox1_MouseDown (object sender, MouseEventArgs e) {
 			osd.BeginInvoke((MethodInvoker)delegate {
 				osd.currentlyselected = getMouseOverItem(e.X, e.Y);
-                adjustGroupbox();
-			});
+                adjustGroupbox();

```

Comment:
I kept it inside the delegate block, but it's not necessary as it wont deal with any UI control.